models/src/main/java/com/vimeo/networking2/enums/ErrorCode.kt

```diff
@@ -0,0 +1,464 @@
+package com.vimeo.networking2.enums
+
+import com.squareup.moshi.Json
+
+/**
+ * All the error codes that can come back from the Vimeo Api.
+ */
+enum class ErrorCode {
```

Comment:
We should have a separate discussion about whether we want to support this `ErrorCode` enum as is because it has caused us problems in the previous design of the library. Examples of those problems:
- Not being able to handle an error until it was added to the library (no error code `Int` property to reference)
- It being an enum isn't all that helpful in practice because the number of codes that can be returned for a specific API call is very limited compared to the scope of the `ErrorCode`. 

I think it's fine to leave here as is for the time being, but we should definitely have a more serious discussion about to handle errors.

Reply:
Good point. I will take note of the issues around `ErrorCode` and leave this as it is for now. The upcoming phase of vimeo networking 2.0 is designing the API requests. It has error handling as part of the phase. Discussion around this issue and possible approach would fit the error handling task.